dashboard/app/models/game.rb

```diff
@@ -266,7 +266,7 @@ def has_i18n?
     !([NETSIM].include? app)
   end
 
-  def use_firebase?
+  def use_datablock_storage?
     [APPLAB, GAMELAB].include? app
```

Comment:
same question about gamelab. I'm having trouble understanding how this change and [this comment](https://github.com/code-dot-org/code-dot-org/pull/56279/files#r1524578742) go together

Reply:
Good question, confusing name. This will all be deleted post-firebase-cleanup, the real bug here is that we don't have it flagged as such! This is only called in one place (in level_helper.rb), and since its not long-term surviving code, I'm going to move it there and just remove this code.

Reply:
We removed this in favor of the version in DatablockStorageController, which now references Game::APPLAB and Game::GAMELAB, sort of giving us the best of bothe worlds